ui/components/app/advanced-gas-inputs/advanced-gas-inputs.component.js

```diff
@@ -140,7 +142,11 @@ export default class AdvancedGasInputs extends Component {
         <div className="advanced-gas-inputs__gas-edit-row__label">
           {label}
           <Tooltip title={tooltipTitle} position="top" arrow>
-            <i className="fa fa-info-circle" />
+            <Icon
+              name={ICON_NAMES.INFO}
+              color={IconColor.iconAlternative}
+              className="info-circle"
+            />
```

Comment:
I think we should use `ButtonIcon` here

Reply:
On second thought I think the tooltip component makes this keyboard accessible. Apologies. I think this is fine.